src/plotting.jl

```diff
@@ -30,81 +30,95 @@ end
 """
 Plot prediction of the GP as a soft decision boundary between [0,1].
 """
-function plot_soft_boundary(gp, models; m=[200,200], show_data=true, overlay=false, overlay_levels=50, ms=4, lw=0, tight=false) # lw=0.2
-    y = gp_output(gp, models; m)
-    model_ranges = get_model_ranges(models, size(y))
+function plot_soft_boundary(gp, models; inds=[:, :, fill(1, length(models) - 2)...], num_steps=200, m=fill(num_steps, length(models)), show_data=true, overlay=false, overlay_levels=50, ms=4, lw=0, tight=false) # lw=0.2
+    y = gp_output(gp, models; m)[inds...]
+    full_models = [model for (model, d) in zip(models, inds) if d == :]
```

Comment:
(nit) `dimension` or `dim` instead of `d`?

Reply:
What is `if d == :`? If dimension is not empty?

Reply:
I re-wrote this a bit to make it more explicit.  An exception will be thrown if there are not exactly two colons in the `inds` argument